doc/service_config.md

```diff
@@ -0,0 +1,110 @@
+All service configs for a particular namespace can be retrieved with the following command:
+```sh
+mobile get clientconfig example_client_id --namespace=myproject
+```
+
+Which will produce output like the following:
+```sh
++----------------+----------------+----------------+-------------------------------------------------------+
+|       ID       |      NAME      |      TYPE      |                          URL                          |
++----------------+----------------+----------------+-------------------------------------------------------+
+| Client ID      | example_client_id         |                |                                                       |
+| fh-sync-server | fh-sync-server | fh-sync-server | https://fh-sync-server-myproject.192.168.64.74.nip.io |
+| keycloak       | keycloak       | keycloak       | https://keycloak-myproject.192.168.64.74.nip.io       |
+| prometheus     | prometheus     | prometheus     | https://prometheus-myproject.192.168.64.74.nip.io     |
++----------------+----------------+----------------+-------------------------------------------------------+
+```
+
+For more verbose config details of each service, you can request the output in JSON format instead, with the following command:
+```sh
+mobile get clientconfig example_client_id --namespace=myproject -o json
+```
+
+Which will produce output similar to the following (newlines and indentation have been added for readability):
+```
+{
+  "version": "1.0",
+  "cluster_name": "192.168.64.74:8443",
+  "namespace": "myproject",
+  "client_id": "example_client_id",
+  "services": [
+    {
+      "id": "fh-sync-server",
+      "name": "fh-sync-server",
+      "type": "fh-sync-server",
+      "url": "https://fh-sync-server-myproject.192.168.64.74.nip.io",
+      "config": {
+        "url": "https://fh-sync-server-myproject.192.168.64.74.nip.io"
+      }
+    },
+    {
+      "id": "keycloak",
+      "name": "keycloak",
+      "type": "keycloak",
+      "url": "https://keycloak-myproject.192.168.64.74.nip.io",
+      "config": {
+        "auth-server-url": "https://keycloak-myproject.192.168.64.74.nip.io/auth",
+        "clientId": "juYAlRlhTyYYmOyszFa",
+        "realm": "myproject",
+        "resource": "juYAlRlhTyYYmOyszFa",
+        "ssl-required": "external",
+        "url": "https://keycloak-myproject.192.168.64.74.nip.io/auth"
+      }
+    },
+    {
+      "id": "prometheus",
+      "name": "prometheus",
+      "type": "prometheus",
+      "url": "https://prometheus-myproject.192.168.64.74.nip.io",
+      "config": {}
+    }
+  ]
+}
+```
+
+### Understanding the JSON format
+Firstly, the parent object in the JSON output is described below:
+
+#### version
+The version of the JSON structure used in this response.
+
+#### cluster_name
+An identifier of the cluster this config was retrieved from.
+
+#### namespace
+The namespace these configs were retrieved from.
+
+#### client_id
+The client id of the mobile application using these configs.
```

Comment:
What is the function of this parameter?

Reply:
it is only part of the JSON response, the CLI does nothing with it currently.

Reply:
Yeah. Actually saw that in code bellow. Works for me!

Reply:
It will be used in the future to allow us to look up the bundleID or package name and send that back as part of the config. This can then be sent along with the mobile client id to verify that a given client app is making the request, it can also be used on start up in the sdk to check against the package of the android app for instance and have the sdk doing something if they don't match.